lib/config.js

```diff
@@ -11,6 +11,7 @@ config.passport.strategies = {};
 
 config.passport.strategies.signature = {};
 config.passport.strategies.signature.disabled = false;
+config.passport.strategies.publicKeyDidLookup = true;
```

Comment:
"publicKeyDidLookup" is not a strategy, "signature" is. This should be moved underneath the "signature" namespace.  We should also change it to look like this:

```js
config.passport.strategies.signature.dereferenceUrlScheme = {
  did: true,
  http: false,
  https: false
};
```

Then the code can check to see if the scheme for the URL is in that config map (and truthy), and if so, it can attempt to dereference the URL. We can add support for HTTPS later.